rpc/wrtc_signaling_answerer.go

```diff
@@ -320,11 +302,25 @@ func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_Answ
 	var successful bool
 	defer func() {
 		if !(successful && err == nil) {
+			candPair, hasCandPair := webrtcPeerConnCandPair(pc)
+			connInfo := getWebRTCPeerConnectionStats(pc)
+			iceConnectionState := pc.ICEConnectionState()
+			iceGatheringState := pc.ICEGatheringState()
+			ans.logger.Infow("Connection establishment failed",
```

Comment:
I could buy into the failure log specifically being a `Warn`

Reply:
Good point; switched to `Warn`.